docs/azure_arc_jumpstart/azure_arc_vsphere/resource_bridge/powershell/_index.md

```diff
@@ -0,0 +1,197 @@
+
+- A resource pool or a cluster with a minimum capacity of 16 GB of RAM and four vCPUs.
+
+- A datastore with a minimum of 100 GB of free disk space available through the resource pool or cluster.
+
+As mentioned, this guide starts at the point where you already have an up and running VMware environment managed by vCenter. The automation will be run from a PowerShell window on a computer (can be your local computer) that has network connectivity to vCenter.
+
+    > **NOTE: the script will automatically uninstall any pre-existing Azure CLI versions in the workstation and will deploy the latest 64 bit version, as it is a requirement to deploy the Resource Bridge**
+
+## Automation Flow
+
+For you to get familiar with the automation and deployment flow, below is an explanation.
+
+- User is editing the onboarding PowerShell script to match the environment (1-time edit).
+
+- User will run the script from the local workstation.
+
+- User will verify the correct onboarding.
+
+## Connect VMware vCenter Server to Azure Arc
+
+- Change the environment variables according to your environment:
+  - _'location'_ - the Azure Region you want to deploy to
+  - _'SubscriptionId'_ - your subscription ID
+  - _'ResourceGroupName'_ - name of the Azure Resource Group you will create your resources in
+  - _'applianceName'_ - a name for the Bridge appliance
+  - _'customLocationName'_ - a name for the Azure Arc custom location
+  - _'vCenterName'_ - your vCenter name
+  - _'vcenterfqdn'_ - your vCenter fully qualified name
+  - _'vcenterusername'_ - username to authenticate to vCenter
+  - _'vcenterpassword'_ - password to authenticate to vCenter
+  - _'appID'_ - your service principal App ID
+  - _'password'_ - your service principal password
+  - _'tenantId'_ - your Azure AD tenant ID
+  - _'vmtemplate'_ - the Arc appliance template name
+  - _'datacenter'_ - the vSphere data center where the appliance will be deployed
+  - _'datastore'_ - the vSphere data store name where the appliance will be deployed
+  - _'folder'_ - the vSphere folder where the template and appliance will be deployed
+  - _'dnsserver'_ - DNS server to be used for the appliance
+  - _'gateway'_ - Gateway address to be used for the appliance
+  - _'ipaddressprefix'_ - Network address in CIDR notation to be used by the appliance
+  - _'k8snodeippoolstart'_ - IP range start for the IPs to be used by the appliance
+  - _'k8snodeippoolend'_ - IP range end for the IPs to be used by the appliance
+  - _'segment'_ - Name of the virtual network or segment to which the appliance VM must be connected
+  - _'resourcepool'_ - the name of the resource pool to be used by the appliance
+  - _'controlPlaneEndpoint'_ - IP address of the Kubernetes cluster control plane
+  - _'vSphereRP'_ - ConnectedVMwarevSphere resource provider Id 
+  
+  ![Screenshot environment variables](./02.png)
```

Comment:
See comment on the naming convention. Also, make sure it reflects in the screenshot